examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -0,0 +1,135 @@
+
+## 2. Create your code and unit tests
+Creating unit tests is beyond the scope of this README, but if you review the tests in tests/ you'll quickly get the idea. 
+I'm using pytest as the testing suite for this project. Before proceeding make sure you can run your tests from the 
+command line
+
+```
+python -m pytest 
+
+```
+
+Or if you want to be fancy and use the coverage plug-in
+
+```
+python -m pytest --cov=my_module tests/
+
+```
+
+If everything goes well you should expect to see output like this, showing successful tests:
+
+```
+$ python -m pytest --cov=my_module tests/
+============================================================================= test session starts ==============================================================================
+platform darwin -- Python 3.7.3, pytest-4.6.2, py-1.8.0, pluggy-0.12.0
+rootdir: /Users/mikebernico/dev/basic-cicd-cloudbuild
+plugins: cov-2.7.1
+collected 6 items
+
+tests/test_my_module.py ......                                                                                                                                           [100%]
+
+---------- coverage: platform darwin, python 3.7.3-final-0 -----------
+Name                     Stmts   Miss  Cover
+--------------------------------------------
+my_module/__init__.py        1      0   100%
+my_module/my_module.py       4      0   100%
+--------------------------------------------
+TOTAL                        5      0   100%
+
+
+```
+
+Now that our tests are working locally, we want to configure GCP to run them every time we push new code.
+
+## 3. Building a Python Cloud Builder
+In order to execute python based tests in cloud builder you need a python cloud builder. 
+A cloud builder is just a docker container with the stuff you need in it. Google doesn't distribute a 
+python cloud builder, so we have to build one ourselves. I've created one in '/python-cloud-builder.' All you need to do 
+is run this:
+
+```
+cd python-cloud-builder
+gcloud builds submit --config=cloudbuild.yaml .
+
+```
+
+This creates a custom cloud builder in your container registry called gcr.io/$PROJECT_ID/python-cloudbuild that you can
+use to run tests with.
+
+## 4. Create a Cloud Build Trigger
+* On the GCP console navigate to 'Cloud Build' > 'Triggers'
+* Add a trigger
+* Choose cloud source repository
+* Pick the repo you created in step 1.
+* Assuming you want the trigger to fire on any branch, accept the default trigger type and regex
+* Choose 'cloud build configuration file' from 'build configuration'
+* Click create trigger
+
+
+## 5. Create a cloudbuild.yaml file that executes your tests and runs pylint
+Create a file called cloudbuild.yaml in the root of your project and add the steps needed to execute your unit tests, 
+using the cloud builder we created in step 3. 
+
+*Note: Each cloudbuilder step is a separate, ephemerial run of a docker container, however the /workspace/ directory is
```

Comment:
Can you link to some info in the docs here?

Reply:
Added.